tests/suites/test_suite_x509parse.data

```diff
@@ -1326,9 +1326,9 @@ X509 CRT ASN1 (TBS, inv SubPubKeyInfo, empty
 depends_on:MBEDTLS_RSA_C:MBEDTLS_SHA256_C
 x509parse_crt:"306b3056a0030201008204deadbeef300d06092a864886f70d01010b0500300c310a30080600130454657374301c170c303930313031303030303030170c303931323331323335393539300c310a300806001304546573743000300d06092a864886f70d01010b0500030200ff":"":MBEDTLS_ERR_PK_INVALID_ALG + MBEDTLS_ERR_ASN1_OUT_OF_DATA
 
-X509 CRT ASN1 (TBS, inv SubPubKeyInfo, inv algorithm tag
+X509 CRT ASN1 (TBS, inv SubPubKeyInfo, inv algorithm tag)
```

Comment:
Same here (move to commit "Fix test case name formatting in X.509 parsing suite")

Reply:
Fixed.